pkg/gpu/testutil/docker.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build test
```

Comment:
should this also have the linux_bpf tag as other testutil files?

Reply:
I don't think it's needed, it might be useful in case code in other packages wants to use it.